logging/src/Logger.cpp

```diff
@@ -34,3 +34,4 @@ logging::Logger ST_SRC_LOG = logging::getLogger("concord.bft.st.src");
 logging::Logger MSGS = logging::getLogger("concord.bft.msgs");
 logging::Logger CL_MNGR = logging::getLogger("concord.bft.client-manager");
 logging::Logger TS_MNGR = logging::getLogger("concord.bft.time-service-manager");
+logging::Logger ON_CHAIN_LOG = logging::getLogger("persist.on-chain");
```

Comment:
Isn't it logically belongs to a "concord" logging namespace?

Reply:
it's cross layers feature from bft to concord, what do you think?

Reply:
all our logs start from "concord".
besides, why key-exchange log is not suitable here?

Reply:
easier to analyse, but I can reuse it